src/urllib3/connectionpool.py

```diff
@@ -558,7 +558,7 @@ def urlopen(  # type: ignore[override]
         .. note::
 
            More commonly, it's appropriate to use a convenience method provided
-           by :class:`.RequestMethods`, such as :meth:`request`.
+           by :class:`urllib3._request_methods.RequestMethods`, such as :meth:`request`.
```

Comment:
Can you please remove "provided by urllib3._request_methods.RequestMethods"? It's a private API, and we already link to urllib3.request().